lib/bolt/bolt_option_parser.rb

```diff
@@ -743,11 +743,15 @@ def initialize(options)
              "Specify a default transport: #{TRANSPORTS.keys.join(', ')}") do |t|
         @options[:transport] = t
       end
-      define('--ssh-command EXEC', "Executable to use instead of the net-ssh ruby library. ",
+      define('--[no-]native-ssh', 'Whether to shell out to native SSH or use the net-ssh Ruby library.',
+             'This option is experimental') do |bool|
+        @options[:'native-ssh'] = bool
+      end
```

Comment:
This option doesn't get picked up by the config class since `Bolt::Config#normalize_overrides` pulls out the relevant overrides using the `OPTIONS` hash: https://github.com/puppetlabs/bolt/blob/master/lib/bolt/config.rb#L250-L253

Reply:
It should - I modified `options` in the SSH config class to include ssh-command and native-ssh. They won't be documented because they aren't in OPTIONS.

Reply:
Ah, I need to add 'native-ssh' there